roles/fluentd_master/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+# TODO: Update fluentd install and configuration when packaging is complete
+- name: download and install td-agent
+  yum:
+    name: 'http://packages.treasuredata.com/2/redhat/7/x86_64/td-agent-2.2.0-0.x86_64.rpm'
+    state: present
+
+- name: install Kubernetes fluentd plugin
+  command: '/opt/td-agent/embedded/bin/gem install fluent-plugin-kubernetes'
```

Comment:
oof. 

I like the todo above, but yeah, this is pretty painful.

Can you at least check if it's installed or something so we don't try to do this every single config run?